web-app/CKS.Web.Test/IntegrationTests/404Tests.cs

```diff
@@ -0,0 +1,28 @@
+using CKS.Web.Test.IntegrationTests.Infrastructure;
+using Shouldly;
+using System;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace CKS.Web.Test.IntegrationTests
+{
+	public class NotFoundTests : TestBase
+	{
+		public NotFoundTests(ITestOutputHelper output)
+			: base(output)
+		{
+		}
+
+		[Fact]
+		public async void Returns404Page()
```

Comment:
Maybe something like this would be a better name:
```suggestion
		public async void ReturnsCustom404PageForUnmatchedRoute()
```